app/client/components/user/CreateUserForm.tsx

```diff
@@ -47,6 +65,10 @@ const validate = (values: ICreateUser) => {
 };
 
 class CreateUserForm extends React.Component<InjectedFormProps<ICreateUser, Props> & Props> {
+  componentDidMount() {
+    this.props.initialize({ accountType: TransferAccountTypes.USER.toLowerCase() });
```

Comment:
I don't know very much about TS, but this feels like a weird coercion of types to a string...

Reply:
need to force lowercase as the value is set like that in the form select field component

https://github.com/teamsempo/SempoBlockchain/blob/5bbbd3e9a3e73fb6f99f427b9236c92d1edd1f46/app/client/components/form/SelectField.tsx#L24